model/clusters_mgmt/v1/oidc_config_resource.model

```diff
@@ -15,13 +15,18 @@ limitations under the License.
 */
 
 // Manages a hosted oidc configuration.
```

Comment:
Is this still always a `hosted` oidc configuration?

Reply:
Oh missed it somewhere